applications/plugins/image/examples/ImageTransform.scn

```diff
@@ -1,4 +1,3 @@
-<!-- example of how to give translation parameter to an image and update it every time step-->
```

Comment:
According to various XML validators, indeed a XML scene should always start with the `<?xml....` stuff.
But I dont think deleting the comment like a brute is a good way to fix 😅

Move the comment below after the  `<?xml....` line and it should work (at least according to https://www.w3schools.com/xml/xml_validator.asp

Reply:
Sorry, did it to quickly...